index.html

```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="utf-8" />
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<title>Page Title</title>
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+	<link rel="stylesheet" href="css/style.css" />
+	<base target="_blank">
```

Comment:
It is redundant. It will be applied for all links on the page, it's better to add `target` attribute only for anchor tag `<a target="_blank">` which you want to opens in new tab.